static/extensions/theAYscratch/Literals

```diff
@@ -0,0 +1,336 @@
+        text: `'[TEXT]'`,
+        arguments: {
+            "TEXT": {
+                type: Scratch.ArgumentType.STRING,
+            },
+        },
+        disableMonitor: true
+    });
+    Extension.prototype[`text`] = async (args, util) => {
+        return args["TEXT"]
+    };
+
+    blocks.push({
+        opcode: `bool`,
+        blockType: Scratch.BlockType.REPORTER,
+        text: `<[BOOLEAN]>`,
+        arguments: {
+            "BOOLEAN": {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: true,
+            },
+        },
+        disableMonitor: true
+    });
+    Extension.prototype[`bool`] = async (args, util) => {
+        return args["BOOLEAN"]
+    };
+
+    blocks.push({
+        opcode: `num`,
+        blockType: Scratch.BlockType.REPORTER,
+        text: `([NUMBER])`,
+        arguments: {
+            "NUMBER": {
+                type: Scratch.ArgumentType.NUMBER,
+            },
+        },
+        disableMonitor: true
+    });
+    Extension.prototype[`num`] = async (args, util) => {
+        return args["NUMBER"]
+    };
+
+    blocks.push({
+        opcode: `color`,
+        blockType: Scratch.BlockType.REPORTER,
+        text: `[COLOR]`,
+        arguments: {
+            "COLOR": {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: '#aa239c',
+            },
+        },
+        disableMonitor: true
+    });
+    Extension.prototype[`color`] = async (args, util) => {
+        return args["COLOR"]
+    };
+
+    blocks.push({
+        opcode: `costume`,
+        blockType: Scratch.BlockType.REPORTER,
+        text: `c:[COSTUME]:ↄ`,
+        arguments: {
+            "COSTUME": {
+                type: Scratch.ArgumentType.COSTUME,
+            },
+        },
+        disableMonitor: true
+    });
+    Extension.prototype[`costume`] = async (args, util) => {
+        return args["COSTUME"]
+    };
+
+    blocks.push({
+        opcode: `sound`,
+        blockType: Scratch.BlockType.REPORTER,
+        text: `s:[SOUND]:ƨ`,
+        arguments: {
+            "SOUND": {
+                type: Scratch.ArgumentType.SOUND,
+            },
+        },
+        disableMonitor: true
+    });
+    Extension.prototype[`sound`] = async (args, util) => {
+        return args["SOUND"]
+    };
+
+    blocks.push({
+        opcode: `angle`,
+        blockType: Scratch.BlockType.REPORTER,
+        text: `s:[ANGLE]:ƨ`,
+        arguments: {
+            "ANGLE": {
+                type: Scratch.ArgumentType.ANGLE,
+            },
+        },
+        disableMonitor: true
+    });
+    Extension.prototype[`angle`] = async (args, util) => {
+        return args["ANGLE"]
+    };
+
+    blocks.push({
+        opcode: `matrix`,
+        blockType: Scratch.BlockType.REPORTER,
+        text: `01 [MATRIX] 10`,
+        arguments: {
+            "MATRIX": {
+                type: Scratch.ArgumentType.MATRIX,
+                defaultValue: 1.00110101001111e+22,
```

Comment:
use `'000011110101100101010110'` as the matrix field can't handle a real number and requires a string containing the binary sequence